docs/using-nativeaot/README.md

```diff
@@ -1,5 +1,6 @@
 # Using Native AOT
 
+- [Building](../workflow/building/coreclr/nativeaot.md)
```

Comment:
People will get to this doc by clicking on "how to debug, optimize and troubleshoot applications published using the native AOT toolchain" in the readme.

How to build the repo is not relevant for that.